src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -24,8 +24,8 @@ public String index(Model model){
         ticket.setEmail("larry@test.com");
         ticket.setAssignee("morganaj@uc.edu");
         ticket.setStatus("unassigned");
-        ticket.setFirstname("Larry");
-        ticket.setLastname("Fine");
+        ticket.setFirstName("Larry");
+        ticket.setLastName("Fine");
```

Comment:
Naming conventions for Java say that methods are mixed case starting with a lowercase letter and the first letter of each following word capitalized.
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html